sdk/src/main/scala/com/horizen/account/api/http/AccountTransactionApiRoute.scala

```diff
@@ -75,7 +76,7 @@ case class AccountTransactionApiRoute(override val settings: RESTApiSettings,
     val secret = allAccounts.find(
       a => (fromAddress.isEmpty ||
         BytesUtils.toHexString(a.asInstanceOf[PrivateKeySecp256k1].publicImage
-          .address) == fromAddress.get) &&
+          .address.toBytes) == fromAddress.get) &&
```

Comment:
check this line: here we compare `byte[]` with `string`

Reply:
no, it is `string == string`